### ISSUE TITLE
Add plugin support

### ISSUE THREAD

**Issue by nfischer** [shelljs/shx#73]
It would be nice if there was support for incorporating ShellJS plugins into shx, like so:

```bash
$ shx open file.txt
$ shx git status # assuming there is a git plugin
$ shx mktemp foo.XXXXXX
```

Since shx blacklists certain commands, it might be good to have some option when creating the ShellJS plugin about whether it will be exposed to `shx`. This would also get rid of the need for a literal blacklist, since it would then be redundant.

**Comment by ariporad** [comment]
Agreed. We could just require everything we can with a module like [`load-plugins`](https://www.npmjs.com/package/load-plugins).

**Comment by nfischer** [comment]
Will that module work with ShellJS plugins? It looks like it's designed for a few specific task runners.

**Comment by ariporad** [comment]
I think so, AFAICT it just uses a wildcard.

**Comment by nfischer** [comment]
Fixed by #78
